enhancements/sha-bundle-digest.md

```diff
@@ -0,0 +1,80 @@
+
+---
+title: SHA-Bundle-Digest
+authors:
+- "@theishshah"
+reviewers:
+- @camilamacedo86 
+- @joelanford 
+- @jmrodri
+- @ryantking 
+- @tlwu2013 
+approvers:
+- @tlwu2013 
+- @jmrodri
+- @joelanford 
+creation-date: 2022-01-14
+last-updated: 2022-01-14
+status:implementable
+---
+
+# Title
+Add support for SHA-based Digest for Images
+
+## Summary
+
+Currently, operators generated by `operator-sdk` will store and retrieve container images based on the image name and the image tag in the bundle. This EP adds support to `operator-sdk` such that images can be stored and retrieved in the bundle with an image name and an image digest as an optional replacement for a tag. Image digests are SHA-256 sums that are hashed from the image layer therefore uniquely identifying one specific image.
```

Comment:
Supreme nit, but can you blast this doc with prettier? Would be nice to have consistent formatting, especially line widths just to make reading easier and remove the need to review whitespace/fromatting things.

```
$ prettier --parser markdown --print-width 120 --single-quote --write enhancements/sha-bundle-digest.md
```

That should do the trick.